pkg/networkmanager/firewall.jsx

```diff
@@ -777,7 +800,40 @@ export class Firewall extends React.Component {
     }
 
     onRemoveZone(zone) {
-        firewall.deactiveateZone(zone);
+        console.log();
+        let body;
+        if (firewall.zones[zone].services.indexOf("cockpit") !== -1)
+            body = <Alert type="warning">{_("This zone contains the cockpit service, make sure that it doesn't apply to the connection used to reach cockpit.")}</Alert>;
```

Comment:
Need some kind of connection to Web Console here, since that's what we'll call it everywhere else. I think it's OK to say the cockpit service, but I would add something about the web console.
Maybe something like:
_This zone contains the cockpit service. Removing the zone risks making the web console unreachable._